### DISCUSSION TITLE
Show more details in the leaderboard shown while playing a map.

### DISCUSSION THREAD

**Discussion by kambojk** [ppy/osu#27868] [Ideas] [answered]
Hey,

I wanted to post a discussion about something that I really value when I play osu. I'd like to see more details in the leaderboard on the left side during gameplay.

One of my favorite things to do in stable is to take a quick glance at the scores next to my current score while I'm playing the map. I know the details expand when there is a break in the map, but I'd like the option to see the detailed view of the score at all times. I'd like to see something like this kind of view while playing the map:
![osu_2024-04-15_02-12-57](https://github.com/ppy/osu/assets/35246586/95eada84-777e-401e-9729-d8d00bf81adb)

In that screenshot, you can only see that view of the leaderboard when you're paused, in a break, or starting the map. I also want to see my placement on the leaderboard that I was viewing before I picked the map. If I was viewing the friend leaderboard before playing the map, I'd like to see all of my friends scores on the side.  I would like this to also apply to playing multis with friends.

I was searching for any discussions related to this, but I couldn't find one. Apologies if this has been asked for before. If this actually exists as a setting and I just didn't see it, then let me know. 

Thank you!

**Comment by SupDos** [comment, marked as answer]
Has been asked previously (https://github.com/ppy/osu/discussions/22040), however as per https://github.com/ppy/osu/pull/24947 and https://github.com/ppy/osu/pull/27285 they want to make the leaderboard a skinnable element (as in you can select/move it in the skin editor) and add a setting to it there to always have it expanded instead of adding a toggle to the settings page, so it won't be a thing until that happens.

**Reply by kambojk** [reply]
Oh, I see. Thank you so much for the discussion/PRs! I can go ahead and mark this discussion as a duplicate of #22040.
